content/en/integrations/cloud_foundry.md

```diff
@@ -153,6 +153,14 @@ cf set-env app01 STD_LOG_COLLECTION_PORT 10514
 cf set-env app01 LOGS_CONFIG '[{"type":"tcp","port":"10514","source":"java","service":"app01"}]'
 ```
 
+**Notification in case of misconfigured proxy**
+
+When using a [proxy configuration](https://docs.datadoghq.com/agent/proxy/?tab=agentv6#proxy-for-logs) with the Buildpack to send your logs to Datadog, a verification is made since Agent 6.12 to check if the connection can be established and depending on the result of this test to start the Log collection.
```

Comment:
```suggestion
For Agent v6.12+, when using a [proxy configuration][27] with the Buildpack, a verification is made to check if the connection can be established. Log collection is started depending on the result of this test.
```
Add the following to the bottom of the page:
`[27]: https://docs.datadoghq.com/agent/proxy/#proxy-for-logs`